sound/soc/sof/intel/hda-dsp.c

```diff
@@ -423,25 +423,19 @@ int hda_dsp_suspend(struct snd_sof_dev *sdev)
 
 int hda_dsp_set_hw_params_upon_resume(struct snd_sof_dev *sdev)
 {
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA)
 	struct hdac_bus *bus = sof_to_bus(sdev);
-	struct sof_intel_hda_stream *hda_stream;
 	struct hdac_ext_stream *stream;
 	struct hdac_stream *s;
-
-#if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA)
 	struct snd_soc_pcm_runtime *rtd;
 	struct hdac_ext_link *link;
 	const char *name;
 	int stream_tag;
-#endif
```

Comment:
please also refine the layout of variable declaration.

Reply:
@rander Ack, will do in next patch revision.